src/applications/personalization/dashboard-2/components/apply-for-benefits/ApplyForBenefits.jsx

```diff
@@ -168,11 +171,16 @@ const ApplyForBenefits = ({
 };
 
 const mapStateToProps = state => {
+  const hasHCAInProgress =
+    selectProfile(state).savedForms?.some(
+      savedForm => savedForm.form === VA_FORM_IDS.FORM_10_10EZ,
+    ) ?? false;
+
   const isPatient = isVAPatient(state);
   const esrEnrollmentStatus = selectESRStatus(state).enrollmentStatus;
 
-  const shouldGetESRStatus = !isPatient && isLOA3(state);
-  const shouldGetDD4EDUStatus = isMultifactorEnabled(state);
+  const shouldGetESRStatus = !hasHCAInProgress && !isPatient && isLOA3(state);
+  const shouldGetDD4EDUStatus = isLOA3(state) && isMultifactorEnabled(state);
```

Comment:
We don't _really_ need to add this additional LOA3 check. But, might as well be extra safe before we start making API calls that have anything to do with financial info.